jsonschema-validation.xml

```diff
@@ -1316,11 +1316,11 @@
             &RFC6901;
             &RFC8259;
             <reference anchor="ecma262"
-            target="http://www.ecma-international.org/publications/files/ECMA-ST/Ecma-262.pdf">
+            target="https://www.ecma-international.org/ecma-262/11.0">
                 <front>
-                    <title>ECMA 262 specification</title>
+                    <title>ECMA-262 11.0 edition specification</title>
```

Comment:
I'd change to "ECMA-262, 11th edition". Or, alternatively, "ECMAScript® 2020 Language Specification, 11th edition".

That's the title they use at the top, and also how they phrase "editions".